Add Optimism bridge box blog post (#1260)
trufflesuite/trufflesuite.com: src/blog/introducing-the-optimism-bridge-truffle-box/index.md

Past L2 Truffle Boxes for [Optimism](https://trufflesuite.com/boxes/optimism) and  [Arbitrium](https://trufflesuite.com/boxes/arbitrum) were built without support for bridging in mind. In this new iteration of Truffle Boxes, we’re focusing on demonstrating bridging support for popular L2 networks.

### Requirements and Installation
As with every Truffle Box, you need to fulfill the following requirements before you can use them:

- [Node.js](https://nodejs.org/) 10.x or later
- [NPM](https://docs.npmjs.com/cli/) 5.2 or later
- [Docker](https://docs.docker.com/get-docker/) 19.03.12 or later
- [Docker-compose](https://docs.docker.com/compose/install) version 1.27.3 or later
- Recommended Docker memory allocation of >= 8 GB
- Windows, Linux, or macOS

Other important, but optional requirements are:
- An [Infura](https://blog.infura.io/?utm_source=blog&utm_medium=pressrelease&utm_campaign=2022_Jun_introducing-the-optimism-bridge-truffle-box_acquisition_content) account and a Project ID

> - A [MetaMask](https://metamask.io/) account

Reviewer: can you do the same for metamask?

1. An [L1 contract](https://github.com/truffle-box/optimism-bridge-box/blob/232a6db7784ed9131d5584f0fc16abec92b21a51/contracts/ethereum/GreeterL1.sol) that sends a message over the Optimism bridge.
2. A [Migration](https://github.com/truffle-box/optimism-bridge-box/blob/232a6db7784ed9131d5584f0fc16abec92b21a51/migrations/3_set_L2_greeting.js) that sends a message from Ethereum to Optimism.
3. An [L2 contract](https://github.com/truffle-box/optimism-bridge-box/blob/232a6db7784ed9131d5584f0fc16abec92b21a51/contracts/optimism/GreeterL2.sol) that sends a message over the Optism bridge.
4. A [Migration](https://github.com/truffle-box/optimism-bridge-box/blob/232a6db7784ed9131d5584f0fc16abec92b21a51/migrations/4_set_L1_greeting.js) that sends a message from Optimism to Ethereum.
5. A [script](https://github.com/truffle-box/optimism-bridge-box/blob/232a6db7784ed9131d5584f0fc16abec92b21a51/scripts/deploy.mjs) to automate the process of compiling contracts and running migrations across each network.

## Conclusion

> That’s all you need to get started building dapps that bridge between Optimism and Ethereum. We’ve put together a short [demo](https://github.com/truffle-box/optimism-bridge-box/blob/232a6db7784ed9131d5584f0fc16abec92b21a51/README.md#demo) that further expands on how the Optimism Bridge Box can be used to bridge networks. We’re excited and can not wait to see what our developer community builds with this these tools.  

Reviewer: ```suggestion
That’s all you need to get started building dapps that bridge between Optimism and Ethereum. We’ve put together a short [demo](https://github.com/truffle-box/optimism-bridge-box/blob/232a6db7784ed9131d5584f0fc16abec92b21a51/README.md#demo) that further expands on how the Optimism Bridge Box can be used to bridge networks. We’re so excited to see what our developer community builds with this these tools!
```

> To find out more about our suite of developer tools, visit the official [Truffle website](https://trufflesuite.com). If you have questions that you'd like for our team members to answer, concerning the merge and how it impacts our products, feel free to start a discussion on our [Github Discussions](https://github.com/orgs/trufflesuite/discussions) channel. Finally, don't forget to follow us on [Twitter](https://twitter.com/trufflesuite) for live announcements and updates.

Reviewer: ```suggestion
To find out more about our suite of developer tools, visit the official [Truffle website](https://trufflesuite.com). If you have questions that you'd like for our team members to answer, feel free to start a discussion on [Github Discussions](https://github.com/orgs/trufflesuite/discussions). Finally, don't forget to follow us on [Twitter](https://twitter.com/trufflesuite) for live announcements and updates.
```
